python/flexflow/keras/models/tensor.py

```diff
@@ -125,3 +127,21 @@ def __verify_ffhandle_shape(self):
 
   def __verify_ffhandle_dtype(self):
     assert self.dtype == self._ffhandle.data_type
+
+  def __add__(self, other):
+    return keras.layers.Add()([self, other])
+
+  def __sub__(self, other):
+    return keras.layers.Subtract()([self, other])
+
+  def __mul__(self, other):
+    return keras.layers.Multiply()([self, other])
+
+  def __matmul__(self, other):
+    return keras.backend.batch_dot(self, other)
+
+  def __div__(self, other):
```

Comment:
Is this okay?

Reply:
I don't have any objection. @jiazhihao ?

Reply:
Edit: I might have quoted the wrong line. I meant is implementing `__div__` as ` keras.layers.Multiply(self, other**-1)` fine?

Reply:
Don't we have `FFModel::divide`?

Reply:
Yes, I had missed that since it wasn't there in Keras. Will add that and update this PR.

Reply:
Have removed `__div__` for now since there's no `keras.layers.Divide` layer (both in Keras and FlexFlow Keras).